modin/pandas/test/test_dataframe.py

```diff
@@ -1735,7 +1735,7 @@ def test_apply(self, request, data, func, axis):
     @pytest.mark.parametrize("axis", [0, 1])
     @pytest.mark.parametrize("level", [None, -1, 0, 1])
     @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-    @pytest.mark.parametrize("func", ["count", "sum", "mean", "all", "kurt"])
```

Comment:
Why were these tests removed?

Reply:
These tests were removed because they were failed because of incorrect level parameter handling, now these tests are added as xfailed.